Expensify/App
# Rename Policy Rooms

cc @TomatoToaster 

### Details
Adds the ability to rename a Policy Room.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/176854

### Tests
1. Create a new policy room by navigating to `+ > New room`.
2. Click the chat header to open up `Details`.
3. Click `Settings`
4. Enter a new value for `Room name` and click `Save`.
5. Verify the following:

- The `Save` button displays a loading spinner and becomes disabled.
- A growl message shows up and reads `Policy room renamed!`.
- A comment like the ones showed in the screenshots below is posted to the chat.
- The room name is updated in the LHN and the chat header.


### QA Steps
Steps above.

- [] Verify that no errors appear in the JS console

### Tested On


### Screenshots


#### Web

![web-pv3](https://user-images.githubusercontent.com/22219519/150401313-f0ff3784-d757-490c-b14d-072f97f0687a.png)

#### Mobile Web

![mweb](https://user-images.githubusercontent.com/22219519/150215155-3de6c938-4d09-4422-bad8-e30659267863.png)

#### Desktop

![desktop](https://user-images.githubusercontent.com/22219519/150215169-015c9dcc-90d3-49a6-885e-598fbe1efc73.png)

#### iOS

![ios-pv3](https://user-images.githubusercontent.com/22219519/150401341-d2574429-b722-4550-b6ae-344d4023a372.png)

#### Android

![android-pv3](https://user-images.githubusercontent.com/22219519/150401362-03121fe0-4b53-4672-9989-6108cd88f1ca.png)
